docs/services/mobilizon.md

```diff
@@ -0,0 +1,58 @@
+# Mobilizon
+
+[Mobilizon](https://joinmobilizon.org/en/) is a ActivityPub/Fediverse server to create and share events here powered by the [mother-of-all-self-hosting/ansible-role-mobilizon](https://github.com/mother-of-all-self-hosting/ansible-role-mobilizon) Ansible role.
+
+## Configuration
+
+To enable this service, add the following configuration to your `vars.yml` file. Also you need to enable postgis which will serve as database for mobilizon.
+After that you can re-run the [installation](../installing.md) process.
+
+```yaml
+########################################################################
+#                                                                      #
+# mobilizon                                                            #
+#                                                                      #
+########################################################################
+
+mobilizon_enabled: true
+
+
+# Hostname that this server will be reachable at.
+# DO NOT change this after your server has already run once, or you will break things!
+mobilizon_hostname: 'events.example.org'
+
+# to open registrations uncomment the following line
+# mobilizon_registrations_open: true
+
+########################################################################
+#                                                                      #
+# /mobilizon                                                           #
+#                                                                      #
+########################################################################
+
+########################################################################
+#                                                                      #
+# postgis                                                              #
+#                                                                      #
+########################################################################
+
+postgis_enabled: true
+
+# Put a strong password below, generated with `pwgen -s 64 1` or in another way
+postgis_connection_password: ''
+
+########################################################################
+#                                                                      #
+# /postgis                                                             #
+#                                                                      #
+########################################################################
```

Comment:
It would probably be better if all this exists only on the `services/postgis.md` page.

For the Mobilizon docs page, you can then list Postgis as a dependency and link ot `services/postgis.md` - just like we do with the other services which have dependencies. Example: https://github.com/mother-of-all-self-hosting/mash-playbook/blob/main/docs/services/authentik.md#dependencies